test/complex.jl

```diff
@@ -135,6 +135,17 @@ end
 @test isinf(complex(0,-Inf))
 @test !isinf(complex(0,0))
 
+# flipsign:
+@test isequal(complex( 0.0, 0.0 ), flipsign(complex( 0.0, 0.0 ), 1))
```

Comment:
FWIW, spaces inside parentheses aren't usually used in the code base, as you can see in the code above.